src/script.js

```diff
@@ -303,7 +303,7 @@ function composeSusiMessage(response, t, rating) {
             });
         }
         storageArr.push(storageObj);
-        chrome.storage.sync.set({
+        chrome.storage.local.set({
```

Comment:
why is this being changed in this `pr?`

Reply:
@zamhaq please read the PR description. Using chrome.storage.local.set.. to set the data instead of chrome.storage.sync.set. As chrome.storage.local.set can contains 5242880 :QUOTA_BYTES.